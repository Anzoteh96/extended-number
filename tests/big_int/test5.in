-6007510093730513
-611149187249870
