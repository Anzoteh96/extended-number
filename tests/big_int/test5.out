Enter two numbers n and m:
n is -6007510093730513
m is -611149187249870
n + m is -6618659280980383
n - m is -5396360906480643
n * m is 3671484911178793364163374283310
n / m is 9
n % m is -507167408481683
